application/controllers/campaign.php

```diff
@@ -1250,7 +1250,7 @@ public function validate($datajson_url = null, $datajson = null, $headers = null
 
         $datajson_url = ($this->input->get_post('datajson_url')) ? $this->input->get_post('datajson_url', TRUE) : $datajson_url;
         $datajson_url = filter_var($datajson_url, FILTER_SANITIZE_URL);
-        $datajson_url = filter_var($datajson_url, FILTER_VALIDATE_URL, FILTER_FLAG_SCHEME_REQUIRED | FILTER_FLAG_HOST_REQUIRED | FILTER_FLAG_PATH_REQUIRED);
+        $datajson_url = filter_var($datajson_url, FILTER_VALIDATE_URL, FILTER_FLAG_PATH_REQUIRED);
```

Comment:
@mogul explained that FILTER_VALIDATE_URL now implicitly uses FILTER_FLAG_SCHEME_REQUIRED and FILTER_FLAG_HOST_REQUIRED https://www.php.net/manual/en/filter.filters.validate.php